lib/util/pdf.js

```diff
@@ -13,7 +13,12 @@ exports.parse = async function parse (docOptions, callbacks) {
     documentParsed: NO_OP,
     ...(callbacks || {}),
   }
-  const pdfDocument = await pdfjs.getDocument(docOptions).promise
+  fontDataPath = __dirname + '/../../../../../node_modules/pdfjs-dist/standard_fonts/'
```

Comment:
you're traversing five levels up from `lib/util`, which means you're going out of the package directory to get to `pdfjs-dist`. Is there a better way to specify the path?